HW02_B_Mimic_starter.Rmd

```diff
@@ -1,99 +1,141 @@
+  theme(plot.title = element_text(hjust = 0.5)) + #center title
+  annotate("text", x = 4, y = 4500, label = "My Best Diamonds,\nof course") + #add caption
+  annotate("rect", xmin = 4.5, xmax = 5.5, ymin = 0, ymax = 5000, alpha = 0.4) #add highlight rect
+  
+
+
+
+
+```
+
+### Graph 2
+```{r, echo = T, include = TRUE}
+data("iris")
+```
+
+Using the iris dataset, make this graph:
+```{r graph 2 code, echo=FALSE}
+iris$SpeciesOrdered <- factor(iris$Species, levels = c("versicolor", "setosa","virginica")) #good order
+
+iris_plot <- ggplot(iris, aes(Sepal.Length, Petal.Length,color = Species, shape = Species))
+
+iris_plot + facet_wrap(vars(SpeciesOrdered),scales = "free") + #make three subplots
+  scale_color_manual(values = c("green","red","dodgerblue2")) + #specify colors
+  scale_shape_manual(values = c(17, 16, 15)) + #specify shapes
+  scale_x_continuous(limits = c(4.2, 8)) + #set xlim
+  geom_point() + #plot points
+  geom_smooth(method = "lm", se = F, color = "black") #plot trend line
+
+```
+
+
+### Graph 3
+You'll need the information in this first box to create the graph
+```{r, echo = TRUE}
+data("mpg")
+corvette <- mpg[mpg$model == "corvette",]
+#install
+require("ggrepel") #useful for making text annotations better, hint hint
+set.seed(42)
+```
+
+Now using the mpg dataset and the corvette dataset, make this graph:
+
+```{r graoh 3 code}
+
+ggplot(corvette, aes(displ,hwy,label = paste("Corvette,",year))) + #plot corvette data
+  labs(title = "Corvettes are a bit of an outlier") + #add title
+  geom_point(color = "blue") + #corvettes are blue
+  geom_text_repel() + #add corvette text
+  geom_point(data = mpg[mpg$model != "corvette",], color = "black") + #plot all other data
+  scale_x_continuous(breaks = c(1:8), limit = c(1,8)) #set xlim and x breaks
+  
+
+
+```
+
+There is a trick to getting the model and year to print off together. `paste()` is a useful function for this, also pasting together parts of file names and parts of urls together. 
+
+
+### Graph 4 
+```{r, echo = T}
+data(mpg)
+
+#hint for the coloring, colorbrewer and you can set palette colors and make your graphs colorblind friendly
+library(RColorBrewer)
+display.brewer.all(colorblindFriendly = T) #take a look at the colorblindfriendly options
+```
+
+The above graph lets you see some colobrlind friendly palettes. For the graph below, I used Set2.
+
+
+
+Now using the above mpg dataset, make this graph
+
+```{r graph 4 code}
+
+ggplot(mpg,aes(class,cty)) + #plot class and city
+  labs(title = "Horizontal BoxPlot of City MPG and Car Class", y = "Car Class", x = "City mpg") + #text
+  geom_boxplot(outlier.shape = NA) + #boxplot, outliers removed
+  coord_flip() + #make horizontal
+  geom_point(aes(color = class), position = position_jitter(width = 0.3, height = 0)) + #points w jit
```

Comment:
This graph looks phenomenal! I shamelessly got inspiration from your code when trying to figure out this problem myself - so thank you!